common/ai_strategies/00_default_strategy.txt

```diff
@@ -1513,6 +1513,57 @@ ai_strategy_default = {
 				}
 				add = 1000
 			}
+			
+			if = {
+				limit = {
+					has_journal_entry = je_tawantisuyu_reconquer_the_empire
+					scope:target_country = {
+						any_scope_state = {
+							OR = {
+								state_region = s:STATE_LIMA
+								state_region = s:STATE_CAJAMARCA
+								state_region = s:STATE_ECUADOR
+								state_region = s:STATE_PASTAZA
+								State_region = s:STATE_CAUCA
+							}
+						}
+					}
+				}
+				add = 1000
+			}
+			
+			if = {
+				limit = {
+					has_journal_entry = je_tawantisuyu_reconquer_the_capital
+					scope:target_country = {
+						any_scope_state = {
+							OR = {
+								state_region = s:STATE_CUZCO
+								state_region = s:STATE_AREQUIPA
+							}
+						}
+					}
+				}
+				add = 1000
+			}
+			
+			if = {
+				limit = {
+					has_journal_entry = je_tawantisuyu_reconquer_the_capital
+					scope:target_country = {
+						any_scope_state = {
+							OR = {
+								state_region = s:STATE_LIMA
+								state_region = s:STATE_CAJAMARCA
+								state_region = s:STATE_ECUADOR
+								state_region = s:STATE_PASTAZA
+								State_region = s:STATE_CAUCA
+							}
+						}
+					}
+				}
+				add = -1000
+			}
 
```

Comment:
Also, add AI strategy to wargoal_scores and bravery.

Reply:
Is there a reason for add = -1000?

Reply:
I'm assuming to make them focus on the capital over the other states while that journal is active, but it is a little redundant, because you already have the focus on the capital in the check above.

Reply:
it adds the -1000
so the ai takes their capital
instead of other provinces
since it is 2 journal entries, that way they try to take their capital first
then the rest
In my testing without this the Incans took 1 capital province and then 1 random other province which was terrible looking